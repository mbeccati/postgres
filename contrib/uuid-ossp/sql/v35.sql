
SELECT uuid_generate_v3(uuid_ns_dns(), 'www.widgets.com');
SELECT uuid_generate_v5(uuid_ns_dns(), 'www.widgets.com');
