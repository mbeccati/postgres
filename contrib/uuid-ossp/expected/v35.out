SELECT uuid_generate_v3(uuid_ns_dns(), 'www.widgets.com');
           uuid_generate_v3           
--------------------------------------
 3d813cbb-47fb-32ba-91df-831e1593ac29
(1 row)

SELECT uuid_generate_v5(uuid_ns_dns(), 'www.widgets.com');
           uuid_generate_v5           
--------------------------------------
 21f7f8de-8051-5b89-8680-0195ef798b6a
(1 row)

